docs/source/contributor-guide/inviting.md

```diff
@@ -0,0 +1,427 @@
+[vote]: https://www.apache.org/foundation/voting
+
+## New Committers
+
+### Step 1: Start a Discussion Thread
+
+The goal of this step is to allow free form discussion prior to calling a vote.
+This helps the other PMC members understand why you are proposing to invite the
+person to become a committer and makes the voting process easier.
+
+Best practice is to include some details about why you are proposing to invite
+the person. Here is an example:
+
+```
+To: private@datafusion.apache.org
+Subject: [DISCUSS] $PERSONS_NAME for Committer
+
+$PERSONS_NAME has been an active contributor to the Datafusion community for the
+last 6 months[1][2], helping others, answering questions, and improving the
+project's code.
+
+Are there any thoughts about inviting $PERSONS_NAME to become a committer?
+
+Thanks,
+Your Name
+
+[1]: https://github.com/apache/datafusion/issues?q=commenter%3A<github id>
+[2]: https://github.com/apache/datafusion/commits?author=<github id>
+```
+
+### Step 2: Formal Vote
+
+Assuming the discussion thread goes well, start a formal vote, with an email like:
+
+```
+To: private@datafusion.apache.org
+Subject: [VOTE] $PERSONS_NAME for DataFusion Committer
+I propose to invite $PERSONS_NAME to be a committer. See discussion here [1].
+
+[ ] +1 : Invite $PERSONS_NAME to become a committer
+[ ] +0: ...
+[ ] -1: I disagree because ...
+
+The vote will be open for at least 48 hours.
+
+My vote: +1
+
+Thanks,
+Your Name
+
+[1] LINK TO DISCUSSION THREAD (e.g. https://lists.apache.org/thread/7rocc026wckknrjt9j6bsqk3z4c0g5yf)
+```
+
+If the vote passes (requires 3 `+1`, and no `-1` voites) , send a result email
+like the following (substitute `N` with the number of `+1` votes)
+
+```
+to: private@datafusion.apache.org
+subject: [RESULT][VOTE] $PERSONS_NAME for DataFusion Committer
+
+The vote carries with N +1 votes.
+```
+
+### Step 3: Send Invitation to the Candidate
+
+Once the vote on `private@` has passed and the `[RESULT]` e-mail sent, send an
+invitation to the new committer and cc: `private@datafusion.apache.org`
+
+In order to be given write access, the committer needs an apache.org account (e.g. `name@apache.org`). To get one they need:
+
+1. An [ICLA] on file. Note that Sending an ICLA to `secretary@apache.org` will trigger account creation. If they already have an ICLA on file, but no Apache account, see instructions below.
+2. Add GitHub username to the account at [id.apache.org](http://id.apache.org/)
+3. Connect [gitbox.apache.org] to their GitHub account
+4. Follow the instructions at [gitbox.apache.org] to link your GitHub account with your
+
+[gitbox.apache.org]: https://gitbox.apache.org
+
+If the new committer is already a committer on another Apache project (so they
+already had an Apache account), The PMC Chair (or an ASF member) simply needs to
+explicitly add them to the roster on the [Whimsy Roster Tool].
+
+### Step 4: Announce and Celebrate the New Committer
```

Comment:
While working on this document it seems like maybe we haven't announced @kazuyukitanimura  or @huaxingao  yet to dev 

@viirya  is that something you would like me to do?

Reply:
Let me use your previous email and send the announcement.

Reply:
Just sent the announcement. Thanks for reminding.